Program.cs

```diff
@@ -1,7 +1,31 @@
 ﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter decimal number: ");
+         decimal.TryParse (Console.ReadLine (), out decimal dec);
+         Console.Write ("Integral Part: ");
+         DeciDigit (dec);
+         Console.Write ("\nFractional Part: ");
+         Fraction (dec);
+      }
+      static void DeciDigit (decimal d) {
+         int whole = (int)d;
+         int[] wholeArr = new int[whole.ToString ().Length];

```

Comment:
Using line 13-17, you could define a new method like "static int[] GetIntArrFromInt(int n) { ... }"

Reply:
Modified.